lifeboat/almalinux/release.multi-arch.sh

```
@@ -33,15 +34,26 @@ docker buildx build --push --platform=linux/arm64,linux/amd64 --tag ghcr.io/dori
                                                               --tag ghcr.io/doridoridoriand/containers/lifeboat-almalinux:8.10-latest -f Dockerfile.8.x .
 
 ##########################
-# 9.4
+# 9.5
 ##########################
 ######### Docker Hub #########
-docker buildx build --push --platform=linux/arm64,linux/amd64 --tag doridoridoriand/lifeboat-almalinux:9.4-$unixtime \
-                                                              --tag doridoridoriand/lifeboat-almalinux:9.4-latest \
+docker buildx build --push --platform=linux/arm64,linux/amd64 --tag doridoridoriand/lifeboat-almalinux:9.5-$unixtime \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)